client/imageCarousels.jsx

```diff
@@ -1,17 +1,36 @@
+/* eslint-disable import/extensions */
 import React from 'react';
 import ReactDOM from 'react-dom';
 import axios from 'axios';
+import ProductView from './components/ProductView.jsx';
 
 class ImageCarousel extends React.Component {
   constructor(props) {
     super(props);
 
-    this.state = {};
+    this.state = {
+      game: {
+        images: [],
+      },
+    };
+  }
+
+  componentDidMount() {
+    axios.get('/api/game')
+      .then((data) => { this.updateGame(data); });
+  }
+
+  updateGame(data) {
+    this.setState({
+      game: data.data,
+    });
```

Comment:
Just out of curiosity, what motivated you to go hooks in your other component and class-based here?

Reply:
So I like using hooks in my components because typically everything I do there is only going to be taking place there and not passed down. Where as on my "main" jsx file I might have to pass down a function or other things, and I'm not quite sure if I'm able to do that while using hooks. I may be able to... Not sure haha.